### ISSUE TITLE
Create FAA DNL graphs for each sensor

### ISSUE THREAD

**Issue by jzabel** [jzabel/aircraftnoiselogger#16]
Just for example data, add second count (   col B) and manipulate timestamp later
Graph 1 hr Sound Exposure dBA
Convert each second dBA to Pa (col D) (=0.00002*10^(dBA at that second/20))
Calculate average Pa over each hour
Convert average Pa to dBA (=20*LOG(average Pa for that hour /0.00002))
For DNL graph, add 10 dBA to all hours   10p-7a
Show DNL for each hour for each sensor. 
Suggestion to do for each day of the week? Not sure if you can average, but at least we can show it by day of the week?

**Comment by jzabel** [comment]
Created sql for this:
SELECT
  EXTRACT(hour FROM CAST(reporting_timestamp AS DATETIME)) AS hour,
  AVG((0.00002*POW(10, db_report/20))) AS pascal
FROM superior-noise-loggers.noise_logging_dataset.noise_logging_observations
WHERE reporting_station = "zabel_flint"
AND EXTRACT(DATE FROM CAST(reporting_timestamp AS DATETIME)) > '2023-09-23'
GROUP BY hour
ORDER by hour

Pull out averages in Pascals and then use excel to add the +10 and graph.

**Comment by jzabel** [comment]
Closing issue. As we looked deeper, we understood you actually can't calculate DNL65. This post from seatac describes it in detail:
IT’S NOT AN EQUATION
https://seatacnoise.info/dnl65-for-dummies/calculating-dnl65/

In other words, DNL65 is not an equation one can ‘calculate’. DNL65 is to airports what the 1040 Schedule C form 165 (depreciation) is for business owners. Which means that it is not a ‘formula’ or a ‘measurement’. It’s one element of a compliance system which is only manipulated by specialist noise consultants–in the same way that only Certified Public Accountants prepare tax returns for large organizations because they are the only ones who know how to work ‘the system’. (For example, the government charges for access to AEDT for the same reason hospitals charge doctors for access to operating rooms. It’s a tool meant only for professionals.)

The process does consist of taking some measurements, with, you know, microphones; but only to build the initial model (and then perhaps to do occasional checks to confirm that the readings are still relevant.)

The measurements are just inputs to the system. It’s the modeling which is the real information used to make the decisions. Let that sink in for a minute.
